Textual updates on markdown files

sexybiggetje opened the pull request:
Does some updates to (a very little number of) typo's, fixes some punctuation and grammar on the root Markdown files.

NyaaaWhatsUpDoc commented:
The typo and spelliing fixes are well appreciated :) though I feel like a lot of this is just changes in choice of wording?

sexybiggetje commented:
> The typo and spelliing fixes are well appreciated :) though I feel like a lot of this is just changes in choice of wording?

I was under the impression it was mostly about . and , :-). But yeah, I replaced a few typos (I think in total 3?), added hyphens where needed. Removed a hyphen. Changed "a bunch of" to the more general accepted "a number of" and related ones.

If there are some specific concerns, I can explain or change those where required. Most of the changes I've made are validated against languagetool.org, first in American English mode (which is the default) but since it caught "favourite" as an error, I've switched it over to British English.

tsmethurst commented:
Some of the phrasing changes are more a matter of taste than correctness, I'd say. But! The typo fixes and stuff are appreciated, and the phrasing changes don't make this *less* readable by any means, so I'm gonna go ahead and merge this. Thank you @sexybiggetje :)
